packages/maskbook/src/plugins/RedPacket/SNSAdaptor/RedPacket/useStyles.ts

```diff
@@ -1,107 +1,141 @@
 import { makeStyles } from '@masknet/theme'
+import { keyframes } from 'tss-react'
```

Comment:
Please reexport keyframes from `@masknet/theme` instead of depends on `tss-react` directly.